### PR TITLE
add TOF diagnostic from ccdb in digitisation (empty crates, TRM errors)

### PR DESCRIPTION
**Pull request by noferini**
Changes in simulation (@chiarazampolli ) to read from ccdb
- offset calibration (to decalibrate)
- diagnostic words -> switch off crates, add TRM errors words

tested with pythia 1000 events@50 kHz  (1 TF) using diagnostic of a run with only 1 crate -> digit QC as expected (matching data QC)

### CODE REVIEW

**Comment by noferini** [comment]
I forgot to say that for the moment to enable ccdb in MC I followed the same philosophy of reconstruction by adding a flag
--use-ccdb-tof

**Comment by noferini** [comment]
all checks have passed

File: DataFormats/Detectors/TOF/include/DataFormatsTOF/Diagnostic.h [hunk 1]
```diff
@@ -49,7 +49,8 @@ class Diagnostic
   void fill(const Diagnostic& diag);                       // for calibration
   void fill(const gsl::span<const o2::tof::Diagnostic>){}; // for calibration
   void merge(const Diagnostic* prev);
-  void getNoisyMap(Bool_t* output); // set true in output channel array
+  void getNoisyMap(Bool_t* output, int noisyThr = 1); // set true in output channel array
+  void getNoisyLevelMap(Char_t* output);              // set true in output channel array
```
Comment (shahor02) [inline DataFormats/Detectors/TOF/include/DataFormatsTOF/Diagnostic.h]: Aren't these const methods?

Comment (noferini) [reply DataFormats/Detectors/TOF/include/DataFormatsTOF/Diagnostic.h]: Yes, they are. Moved to const (including also other non static getters).

**Review by shahor02 — APPROVED** [review]

**Pull request merged**